integration-cli/docker_api_logs_test.go

```diff
@@ -19,34 +19,31 @@ func (s *DockerSuite) TestLogsAPIWithStdout(c *check.C) {
 
 	type logOut struct {
 		out string
-		res *http.Response
 		err error
 	}
+
 	chLog := make(chan logOut)
+	res, body, err := request.Get(fmt.Sprintf("/containers/%s/logs?follow=1&stdout=1&timestamps=1", id))
```

Comment:
nit: Could use the `+` operator for string concatenation instead of `Sprintf`. I find that easier to read and safer against type safety bugs and argument mismatch bugs, but I know now everyone shares the opinion.